.github/workflows/release-scala-models.yml

```diff
@@ -1,39 +0,0 @@
-name: Release to Sonatype
-
-on:
-  release:
-    types: [published]
-
-jobs:
-  release_scala_models:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
-        with:
-          distribution: corretto
-          java-version: 11
-          cache: sbt
-      - name: Release Scala Models
-        run: |
-          VERSION=${{ github.event.release.tag_name }}
-          IS_SNAPSHOT=${{ github.event.release.prerelease }}
-          if [[ ${VERSION:0:1} == "v" ]] ; then
-            VERSION=${VERSION:1}
-          fi
-          if [[ ${VERSION: -9} == "-SNAPSHOT" ]] && [[ $IS_SNAPSHOT == "false" ]] ; then
-            echo "Version must not end in -SNAPSHOT. Exiting."
-            exit 1
-          elif [[ ${VERSION: -9} != "-SNAPSHOT" ]] && [[ $IS_SNAPSHOT == "true" ]] ; then
-            echo "Snapshot version must end in -SNAPSHOT. Adding -SNAPSHOT suffix"
-            VERSION="$VERSION-SNAPSHOT"
-          fi
-          echo $PGP_SECRET | base64 --decode | gpg --batch --import
-          export GPG_TTY=$(tty)
-          echo "Releasing version $VERSION"
-          sbt "clean" "project scalaModels" "set isSnapshot := $IS_SNAPSHOT" "release release-version $VERSION with-defaults"
-        env:
-          PGP_SECRET: ${{ secrets.PGP_SECRET }}
-          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
-          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
TODO: when this change is merged and we've verified the release process we should delete these secrets (and I should also ask for my sonatype account to be removed too)